src/HJSON/HJSONParser.php

```diff
@@ -419,9 +419,8 @@ private function keyname()
 
         while (true) {
             if ($this->ch === ':') {
-                if (!$name && $name != 0) {
-                    $this->error("Found ':' but no key name (for an empty key name use quotes)");
-                } elseif ($space >=0 && $space !== mb_strlen($name)) {
+                if ($name === '') $this->error("Found ':' but no key name (for an empty key name use quotes)");
+                else if ($space >=0 && $space !== mb_strlen($name)) {
```

Comment:
Could you run PHP_CodeSniffer on this again to fix the code merged from master?

Reply:
@dqsully, I've run the `PHP_CodeSniffer` again via latest commit.